observability/logging/logging-6.0/6x-cluster-logging-collector-6.0.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: ASSEMBLY
+:context: cluster-logging-collector-6-1
+[id="cluster-logging-collector-6-1"]
```

Comment:
Should this be? I wouldn't want to mess with any link or bookmarks. But, that don't look right. 
```suggestion
:context: cluster-logging-collector-6-0
[id="cluster-logging-collector-6-0"]
```